CollisionOperator/CollisionProcessors/neutronMGimp_class.f90

```diff
@@ -0,0 +1,371 @@
+module neutronMGimp_class
+
+  use numPrecision
+  use endfConstants
+  use universalVariables,            only : nameWW, maxSplit
+  use genericProcedures,             only : fatalError, rotateVector, numToChar
+  use dictionary_class,              only : dictionary
+  use RNG_class,                     only : RNG
+
+  ! Particle types
+  use particle_class,                only : particle, particleState, printType, P_NEUTRON
+  use particleDungeon_class,         only : particleDungeon
+
+  ! Abstract interface
+  use collisionProcessor_inter,      only : collisionProcessor, collisionData ,init_super => init
+
+  ! Nuclear Data Interface
+  use nuclearDataReg_mod,            only : ndReg_getNeutronMG => getNeutronMG
+  use nuclearDatabase_inter,         only : nuclearDatabase
+  use mgNeutronDatabase_inter,       only : mgNeutronDatabase
+  use mgNeutronMaterial_inter,       only : mgNeutronMaterial, mgNeutronMaterial_CptrCast
+  use reactionHandle_inter,          only : reactionHandle
+  use multiScatterMG_class,          only : multiScatterMG, multiScatterMG_CptrCast
+  use fissionMG_class,               only : fissionMG, fissionMG_TptrCast
+
+  ! Cross section packages
+  use neutronXsPackages_class,       only : neutronMacroXSs
+
+  ! Geometry and fields
+  use geometryReg_mod,                only : gr_fieldIdx => fieldIdx, gr_fieldPtr => fieldPtr
+  use weightWindowsField_class,       only : weightWindowsField, weightWindowsField_TptrCast
+
+
+  ! Nuclear Data
+  !use nuclearData_inter,              only : nuclearData
+  !use perMaterialNuclearDataMG_inter, only : perMaterialNuclearDataMG
+
+  ! Cross-section packages to interface with nuclear data
+  !use xsMacroSet_class,               only : xsMacroSet, xsMacroSet_ptr
```

Comment:
```suggestion
```
I think these pointless commented out lines have been in the file for too long ;-)